package.json

```diff
@@ -99,29 +98,31 @@
     "ember-unused-components": "0.2.0",
     "ember-window-mock": "0.5.4",
     "ember-wormhole": "^0.5.2",
-    "eslint": "^4.19.1",
+    "eslint": "^7.1.0",
     "eslint-config-airbnb-base": "^13.0.0",
-    "eslint-plugin-ember": "^6.2.0",
+    "eslint-plugin-ember": "^8.6.0",
     "eslint-plugin-ember-best-practices": "1.1.1",
     "eslint-plugin-import": "^2.12.0",
-    "eslint-plugin-node": "^9.0.1",
+    "eslint-plugin-node": "^11.1.0",
     "fastboot": "2.0.1",
-    "foundation-sites": "6.5.0-rc.4",
-    "husky": "^6.0.0",
+    "foundation-sites": "6.4.1",
     "labs-ember-search": "^3.1.5",
     "labs-ui": "^0.0.24",
-    "lint-staged": ">=10",
     "loader.js": "^4.7.0",
     "mapbox-gl": "1.0.0",
     "mapbox-gl-draw": "0.16.0",
     "moment": "2.22.2",
-    "prember": "1.0.3",
+    "npm-run-all": "^4.1.5",
+    "prember": "1.1.0",
     "prember-middleware": "0.1.0",
-    "pretender": "3.0.1",
+    "pretender": "^3.4.7",
     "sinon": "7.3.2"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "10.* || >= 12"
```

Comment:
Can you change this to `^12`? I think it would be best to keep these to one major version at a time.